akka-http-core/src/test/scala/akka/http/impl/engine/client/HttpConfigurationSpec.scala

```diff
@@ -137,10 +137,10 @@ class HttpConfigurationSpec extends AkkaSpec {
           akka.http.host-connection-pool.max-connections = 43
         """.stripMargin) { sys ⇒
 
-          val pool = ConnectionPoolSettings(sys)
-          pool.getMinConnections should ===(42)
-          pool.getMaxConnections should ===(43)
-        }
+        val pool = ConnectionPoolSettings(sys)
+        pool.getMinConnections should ===(42)
+        pool.getMaxConnections should ===(43)
+      }
```

Comment:
This (and changes in the files following it) reverts the formatting fixes introduces as part of the sbt-scalariform update.